terraform/modules/sharded_cluster/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "model_name" {
+  description = "Model name"
+  type        = string
+}
+
+variable "config_server_app_name" {
+  description = "config-server app name"
+  type        = string
+  default     = "config-server"
+}
+
+variable "config_server_replicas" {
+  description = "Node count"
+  type        = number
+  default     = 1
```

Comment:
We should perhaps for this one and for the subsequent shards' replicas set the default to 3 as recommended number